Formula/z3.rb

```diff
@@ -21,10 +13,19 @@ class Z3 < Formula
     sha256 "81c89c70da771d0d1faae38657e43849f741cb68486afbf50a21be76bf612799" => :mavericks
   end
 
+  option "without-python", "Build without python 2 support"
+  depends_on :python => :recommended if MacOS.version <= :snow_leopard
+  depends_on :python3 => :optional
+
+  if build.without?("python3") && build.without?("python")
+    odie "z3: --with-python3 must be specified when using --without-python"
+  end
+
   def install
-    inreplace "scripts/mk_util.py", "dist-packages", "site-packages"
+    # This inreplace can be removed on next release.
+    inreplace "scripts/mk_util.py", "dist-packages", "site-packages" if build.stable?
```

Comment:
@UniqMartin thanks for reviewing. My bad, it builds without the `inreplace` but python import path is not set at all. Installation for python bindings is simplified in HEAD build, I'll update the formula a little later.